app/build.gradle.kts

```diff
@@ -155,11 +158,39 @@ dependencies {
     implementation(platform(libs.firebase.bom))
 
     // Adds a remote binary dependency only for local tests.
-    testImplementation(libs.junit)
 
+    testImplementation (libs.mockito.inline)
+    testImplementation (libs.mockito.android)
+    testImplementation ("org.mockito:mockito-core:3.12.4")
+    testImplementation ("org.mockito.kotlin:mockito-kotlin:3.2.0")
```

Comment:
Warning: Having hard-coded version and libs in the build.gradle.kts is bad practice. It would be cleaner if you put them in the libs.version.toml and link them here